scaleway/resource_redis_cluster.go

```diff
@@ -232,27 +316,33 @@ func resourceScalewayRedisClusterUpdate(ctx context.Context, d *schema.ResourceD
 
 	if d.HasChange("name") {
 		req.Name = expandStringPtr(d.Get("name"))
+		changes = append(changes, "name")
```

Comment:
Could you remove those changes in debugging variables? You can use a breakpoint and evaluation to check whether or not a variable passes a given condition.